### DISCUSSION TITLE
disabling a page from breadcrumb

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#3198] [Q&A] [answered]
Please add the option to turn off the page from the breadcrumb through the page frontmatter

**Comment by Mister-Hope** [comment, marked as answer]
Features like this meaning additional data in route and extra size to make the website reactive. Also this should be rare  needed as breadcrumb is supposed to reflect URL structure. We won't support that. If you want it you may override the component and achieve it yourself.
